pom.xml

```diff
@@ -595,6 +596,11 @@
                 <artifactId>poi-ooxml-full</artifactId>
                 <version>${poi.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.javaparser</groupId>
```

Comment:
To be sure that this version is used, you should add an exclusion for javaparser for whereever it originally comes from

Reply:
Excluded from poi-ooxml and defined explicitly